index.js

```diff
@@ -15,9 +16,40 @@ mongoose
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+  .then(x => {
+    if(mongoose.connection.readyState===1) {
+      return Recipe.create({
+        title: 'Hamburger',
+        level: 'Easy Peasy',
+        ingredients: ['steak', 'bread', 'tomatoes', 'cheddar', 'pickles'],
+        cuisine: 'American',
+        dishType: 'main_course',
+        image: 'https://img.cuisineaz.com/660x660/2013/12/20/i9380-hamburger.webp',
+        duration: 60,
+        creator: 'Chef Hadrien',
+        default: Date.now
+      })
+      .then(recipe => console.log(recipe.title))
```

Comment:
You want to `return` the promise and treat it in the next `.then()` otherwise you are making a callback 

```suggestion

```